backend/src/libs/packages/encrypt/encrypt.package.ts

```diff
@@ -10,6 +10,19 @@ class Encrypt implements IEncrypt {
   public async encrypt(password: string, salt: string): Promise<string> {
     return await hash(password, salt);
   }
+
+  public async compare({
+    passwordToCompare,
+    salt,
+    passwordHash,
+  }: {
+    passwordToCompare: string;
+    salt: string;
+    passwordHash: string;
+  }): Promise<boolean> {
+    const hashToCompare = await this.encrypt(passwordToCompare, salt);
+    return hashToCompare === passwordHash;
```

Comment:
Without any spacing dividers it's not the easiest thing to read
```suggestion

    return hashToCompare === passwordHash;
```

Reply:
ok, got it